frontend/packages/topology/src/bindable-services/fetch-bindable-services-utils.ts

```diff
@@ -0,0 +1,48 @@
+import { k8sGet, K8sKind, k8sList } from '@console/internal/module/k8s';
+import { ServiceBindingModel } from '../models';
+import { BindableServicesModel } from './models';
+import { BindableServiceGVK, BindableServicesKind } from './types';
+
+type BindableServicesData = {
+  bindableServices: BindableServiceGVK[];
+  loaded: boolean;
+};
+
+const bindableServicesData: BindableServicesData = {
+  bindableServices: [],
+  loaded: false,
+};
+
+export const fetchBindableServices = async (): Promise<BindableServiceGVK[]> => {
+  try {
+    const bindableService: BindableServicesKind = await k8sGet(
+      BindableServicesModel,
+      'bindable-services',
+    );
+    bindableServicesData.bindableServices = bindableService.spec;
+  } catch (err) {
+    // eslint-disable-next-line no-console
+    console.warn('Error fetching bindable services', err);
+    bindableServicesData.bindableServices = [];
+  }
+  bindableServicesData.loaded = true;
+  return bindableServicesData.bindableServices;
+};
+
+export const getBindableServicesList = async (namespace: string) => {
+  const serviceResources = await Promise.all(
+    bindableServicesData.bindableServices.map((service) => {
+      const serviceKind = {
+        apiGroup: service.group,
+        apiVersion: service.version,
+        kind: service.kind,
+      };
+      return k8sList(serviceKind as K8sKind, { ns: namespace });
+    }),
+  );
+
+  return serviceResources;
+};
+
+export const getServiceBindingsList = (namespace: string) =>
+  k8sList(ServiceBindingModel, { ns: namespace });
```

Comment:
ideally, this should also be fetched through useK8sWatchResource hook in topology